From 39c3d7901afc12cb873cf427924efec40ff597f2 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 17 Jul 2020 19:49:28 +0200
Subject: [PATCH 1/1] dfu: fix typo parameteres

%s/parameteres/parameters/g

Even if a line exceeds 80 characters we should not split output strings
to make debugging easier.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 common/update.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/common/update.c b/common/update.c
index 5ffac3acfe..8abc7bc02e 100644
--- a/common/update.c
+++ b/common/update.c
@@ -170,8 +170,7 @@ got_update_file:
 		printf("\n");
 		if (update_fit_getparams(fit, noffset, &update_addr,
 					&update_fladdr, &update_size)) {
-			printf("Error: can't get update parameteres, "
-								"aborting\n");
+			printf("Error: can't get update parameters, aborting\n");
 			ret = 1;
 			goto next_node;
 		}
-- 
2.27.0

